signal-desktop.sh

```diff
@@ -10,6 +10,12 @@ if (( ${SIGNAL_USE_WAYLAND:-0} )); then
     export ELECTRON_OZONE_PLATFORM_HINT="${ELECTRON_OZONE_PLATFORM_HINT:-auto}"
 fi
 
+SIGNAL_PASSWORD_STORE="${SIGNAL_PASSWORD_STORE:-basic}"
+
+EXTRA_ARGS+=(
+    "--password-store=${SIGNAL_PASSWORD_STORE}"
+)
+
```

Comment:
Given that `--password-store` is unconditionally added to `EXTRA_ARGS`, you could just replace line 3

```bash
EXTRA_ARGS=()
```

with

```bash
EXTRA_ARGS=(
     "--password-store=${SIGNAL_PASSWORD_STORE:-basic}"
)
```